test/jdk/java/awt/print/PrinterJob/ImagePrinting/AlphaPrintingOffsets.java

```diff
@@ -0,0 +1,231 @@
+
+    private static boolean isAlphaTestModeSet = false;
+
+    public static boolean getAlphaTestModeSet() {
+        return isAlphaTestModeSet;
+    }
+
+    public static void main(String[] args) throws Exception {
+        if (PrinterJob.lookupPrintServices().length > 0) {
+
+            String instructionsHeader = "This test prints 6 pages with same image except text messages. \n";
+            if (args.length > 0)
+                isAlphaTestModeSet = args[0].equals("testAlpha");
+            if(isAlphaTestModeSet)
+                instructionsHeader = "This test prints 2 pages with same image except text messages. \n";
+            PassFailJFrame.builder().instructions(instructionsHeader + INSTRUCTIONS)
+                    .testUI(() -> createTestUI()).build().awaitAndCheck();
+
+        } else {
+            System.out.println("Printer not configured or available."
+                    + " Test cannot continue.");
+            throw new SkippedException("Printer not configured or available."
+                    + " Test cannot continue.");
+        }
+
+    }
+
+    public static JFrame createTestUI() {
+        JFrame testUI = new JFrame("Print images");
+        testUI.setSize(1, 1);
+        testUI.addWindowListener(new WindowAdapter() {
+            @Override
+            public void windowOpened(WindowEvent e) {
+                super.windowOpened(e);
+                print();
+            }
+
+            @Override
+            public void windowActivated(WindowEvent e) {
+                super.windowActivated(e);
+                testUI.setVisible(false);
+            }
+        });
+
+        return testUI;
+    }
+
+    private static void print() {
+        PrinterJob printerJob = PrinterJob.getPrinterJob();
+        PageFormat pageFormatP = printerJob.defaultPage();
+
+        Paper paper = pageFormatP.getPaper();
+        paper.setImageableArea(0, 0, paper.getWidth(), paper.getHeight());
+        pageFormatP.setPaper(paper);
+
+        PageFormat pageFormatL = (PageFormat) pageFormatP.clone();
+        PageFormat pageFormatRL = (PageFormat) pageFormatP.clone();
+
+        pageFormatL.setOrientation(LANDSCAPE);
+        pageFormatRL.setOrientation(REVERSE_LANDSCAPE);
+
+        Printable printableOpaque = new CustomPrintable(255);
+        Printable printableTransparent = new CustomPrintable(254);
+
+        Book bookNoAlphaTest = new Book();
+        bookNoAlphaTest.append(printableOpaque, pageFormatP);
+        bookNoAlphaTest.append(printableTransparent, pageFormatP);
+        bookNoAlphaTest.append(printableOpaque, pageFormatL);
+        bookNoAlphaTest.append(printableTransparent, pageFormatL);
+        bookNoAlphaTest.append(printableOpaque, pageFormatRL);
+        bookNoAlphaTest.append(printableTransparent, pageFormatRL);
+
+        Book bookAlphaTest = new Book();
+        bookAlphaTest.append(printableTransparent, pageFormatL);
+        bookAlphaTest.append(printableTransparent, pageFormatRL);
```

Comment:
I guess it can be moved inside if (isAlphaModeSet) and bookNoAlphaTest in else part....
Why to instantiate when we are not going to test bookNoAlphaTest bydefault?